cli/src/action/circuit/mod.rs

```diff
@@ -204,6 +214,49 @@ impl Action for CircuitCreateAction {
     }
 }
 
+#[derive(Deserialize)]
+struct Node {
+    #[serde(alias = "node_id")]
+    identity: String,
+    endpoint: String,
+}
+
+fn load_nodes_from_file(node_file: &str) -> Result<Vec<Node>, CliError> {
+    if node_file.starts_with("http://") || node_file.starts_with("https://") {
+        load_nodes_from_remote(node_file)
+    } else {
```

Comment:
Because we support http://, users may also expect that they can use file://, so it would be good to support that here (just stripping it off if it is given).